### DISCUSSION TITLE
Detect person in one camera, trigger another?

### DISCUSSION THREAD

**Discussion by chrisevans1001** [blakeblackshear/frigate#2936] [Ask A Question] [answered]
I've got a camera looking down across the side of my property. If someone walks into the zone I've set up, it triggers. This is somebody effectively at my front door. I'd like to actually have this event trigger the front door camera as well and I would use this for my notifications.

I don't use detection on my front door camera for notifications as it is on a main road right by a pavement, so anyone and everyone is walking past it every few minutes. There is no zoning as such possible on this camera.

Is there anyway to achieve this?

**Comment by NickM-27** [comment, marked as answer]
Currently there is no way to manually trigger an event from another camera, I have a few thoughts / suggestions

1. I also have a front door camera pointed at a busy street / sidewalk and zones work great, so I'm curious what it is about your setup that makes this impossible? 

2. You could still have it in home assistant where when one camera triggers it sends a notification with a snapshot from the other camera. Just not with a clip (although could use Timestamps to manually generate a clip so that could work as well.

**Comment by chrisevans1001** [comment]
Ah! Now the custom generated timestamp may do the trick as it was the clip I was missing. I had got the snapshot from the other camera but hadn't considered the other option.

Re the set up - it is about 50cm from my front door to the path. The camera on the front door is a doorbell style, so it looks straight out. There is nothing to zone as wherever the people stand is the same place as where people walk.

**Reply by NickM-27** [reply]
Sure, hopefully that works. 

As far as the doorbell camera goes, I am not sure I understand. You may know this but wanted to clarify that zones are triggered based off of the bottom center of the bounding box, not how much the person is inside the zone. 

This is my doorbell camera with zones (excuse the snow), the right zone is front_yard, left is driveway. This setup works perfectly where anyone walking by on the sidewalk are not triggering the zone but people walking even one foot in the yard or driveway will trigger it

![image](https://user-images.githubusercontent.com/14866235/157516112-15b9acb1-eb51-47a7-92f3-c5f217eb5a17.png)

**Comment by chrisevans1001** [comment]
I did realise re the bottom of the bounding box but it just doesn't trigger enough. Our area is simply nowhere near as spacious! :)

I've drawn a black line on yours where our boundary stops. The pavement is immediately where the black line is. So when people ring our doorbell, they're generally standing in the same place as somebody walking along the pavement would be.

![image](https://user-images.githubusercontent.com/4400074/157517020-31af4d90-71db-49ab-9141-c79520af5f18.png)

**Reply by NickM-27** [reply]
Ah I see, that makes sense then. If you have any questions about the clip timestamping feel free to tag me

**Comment by blakeblackshear** [comment]
The only way to do this at the moment is to configure one camera as the detect stream and the other camera as the record stream together. You end up running detection on the same camera twice, but that's the current workaround.

**Comment by nsprawling** [comment]
Would be a really useful feature for my use case - I have a camera that's covering our drive and gets _either_ clear footage of number plates or mediocre nighttime vehicle detections depending on setup. I can't get both from the one cam. If I could have a colour night vision camera covering the drive from a high angle, out of approaching vehicles headlights, this would be a much better detection trigger and get clear images of the vehicles, then I can focus the other on getting good images of plates and hopefully drivers from the current cam.

**Comment by ediguidedog** [comment]
I suggested this https://github.com/blakeblackshear/frigate/discussions/3478 a couple of years back and although it wasn't my use case it would work in this case if you set up a tiny zone in the sky and named it the same as the zone on your detect camera.

**Comment by maxi1134** [comment]
I add that this would be great
